hydra-node/src/Hydra/Snapshot.hs

```diff
@@ -157,7 +180,8 @@ genConfirmedSnapshot headId minSn utxo sks
     -- FIXME: This is another nail in the coffin to our current modeling of
     -- snapshots
     number <- arbitrary `suchThat` (> minSn)
-    let snapshot = Snapshot{headId, number, utxo, confirmed = []}
+    -- TODO: check whether we are fine with this not producing any decommitting utxo ever
```

Comment:
are we suppose to have a spec for this check?

Reply:
This todo is here to make sure we don't forget to check where `genConfirmedSnapshot` is used and if we want to actually test decommits using this function or it is ok to ignore them and leave them always as empty.